QUARKUS-2829.md

```diff
@@ -0,0 +1,55 @@
+# QUARKUS-2829 - Internal support for Micrometer metrics when using managed Vert.x in Quarkus
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-2829
+
+
+As part of supporting Vert.x use-cases on Quarkus we need to support collecting metrics using micrometer for the managed Vert.x instance in Quarkus. This is also a requirement for some layered products like AMQ Streams and OpenShift Serverless that today are using Vert.x, but needs to move to use Quarkus. For 2.13 the support for this is limited to OpenShift Serverless and AMQ Streams, this is not supported for general use. Support for general use should come in RHBQ 3.x
+
+## Scope of testing
+
+Managed Vert.x in Quarkus contains following extensions:
+- Messaging clients:
```

Comment:
The messaging clients are not supported, at least not in RHBQ 2.13.

If we're listing those extensions, maybe it would be useful bit of information they're not supported.

Reply:
`quarkus-smallrye-reactive-messaging-kafka` is supported, so is `quarkus-smallrye-reactive-messaging-amqp`. MQTT is not supported:

https://code.quarkus.redhat.com/?extension-search=origin:platform%20quarkus-smallrye-reactive-messaging-mqtt